openapi3.js

```diff
@@ -40,7 +40,7 @@ function convertToToc(source,data) {
                 method.slug = sMethodUniqueName.toLowerCase().split(' ').join('-'); // TODO {, } and : ?
                 var tagName = data.translations.defaultTag;
                 if (method.operation.tags && method.operation.tags.length > 0) {
-                    tagName = getSectionTag(method.operation.tags[0], data.options.sections);
+                    tagName = getTagGroup(method.operation.tags[0], data.options.sections);
```

Comment:
Sorry to be a total pain, but could we rename `options.sections` too?

Reply:
Done! Good catch, it should have been changed as well.